bootstrap_cfn/fab_tasks.py

```diff
@@ -371,3 +350,13 @@ def rsync():
             env.environment,
             '.'),
         use_sudo=True)
+    tfile_path = '/tmp/{0}-temp-pillar.sls'.format(stack_name)
+    with open(tfile_path, 'w') as tfile:
+        yaml.dump(cfg, tfile) 
+        put(
+            remote_path=os.path.join(
+                remote_pillar_dir,
+                'cloudformation.sls'),
+            local_path=tfile_path,
+            use_sudo=True)
+    os.unlink(tfile_path)
```

Comment:
Do we really have to create a temporary file to do this? Can we pass a StringIO object instead?

> local_path may alternately be a file-like object, such as the result of open('path') or a StringIO instance.